qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -74,13 +158,73 @@ def __init__(
             configuration: Backend configuration.
             api_client: IBM Quantum client used to communicate with the server.
         """
-        super().__init__(configuration=configuration)
-
+        super().__init__(
+            name=configuration.backend_name,
+            online_date=configuration.online_date,
+            backend_version=configuration.backend_version,
+        )
         self._api_client = api_client
-
-        # Attributes used by caching functions.
+        self._configuration = configuration
         self._properties = None
+        self._qubit_properties = None
         self._defaults = None
+        self._target = None
+        self._max_circuits = configuration.max_experiments
+        if not self._configuration.simulator:
+            self.options.set_validator("noise_model", type(None))
+            self.options.set_validator("seed_simulator", type(None))
```

Comment:
Do you want to set validators based on `max_shots`, `rep_delay_range, `qubit_lo_range`, and `meas_lo_range`. Something like:

```suggestion
        if not self._configuration.simulator:
            self.options.set_validator("noise_model", type(None))
            self.options.set_validator("seed_simulator", type(None))
        self.options.set_validator("shots", (1, configuration.max_shots)
        self.options.set_validator(
            "rep_delay", (configuration.rep_delay_range[0], configuration.configuration.rep_delay_range[1])
        )
        self.options.set_validator(
            "qubit_lo_freq", (configuration.qubit_lo_range[0], configuration.configuration.qubit_lo_range[1])
        )
        self.options.set_validator(
            "measure_lo_freq", (configuration.measure_lo_range[0], configuration.configuration.measure_lo_range[1])
        )
```